src/ui/components/ui/qRCode/QrCode.tsx

```diff
@@ -43,7 +43,12 @@ const QrCodeImage = ({
       {isSvg ? (
         <div dangerouslySetInnerHTML={svgSource} />
       ) : (
-        <img srcSet={src} width={options.width} height={options.height} />
+        <img
+          className={!src && 'skeleton-glow'}
```

Comment:
I guess it's better to use ternary, otherwise `false` would end up in `className`